bitnami/kafka/values.yaml

```diff
@@ -1698,13 +1701,43 @@ provisioning:
   ##
   waitForKafka: true
 
+## @section Kraft chart parameters
+
+## Kraft configuration
+## Kafka mode without Zookeeper. Kafka nodes can work as controllers in this mode.
+##
+kraft:
+  ## @param kraft.enabled Switch to enable or disable the Kraft mode for Kafka
+  ##
+  enabled: false
+  ## @param kraft.controllerPort Kafka Controller listener port
+  ##
+  controllerPort: 9095
+  ## @param kraft.processRoles Roles of your Kafka nodes. Nodes can have 'broker', 'controller' roles or both of them.
+  ##
+  processRoles: broker,controller
+  ## @param kraft.controllerListenerNames Controller listener names
+  ##
+  controllerListenerNames: CONTROLLER
+  ## @param kraft.clusterId Kafka ClusterID. You must set it if your cluster contains more than one node.
+  ## Generate with `cat /proc/sys/kernel/random/uuid | tr -d '-' | base64 | cut -b 1-22`. Run `export LC_ALL=C` before if you generate it on MacOS.
+  ## Example: k2yipv1sRue7z2_Y3o976A
+  ##
+  clusterId: ""
```

Comment:
I decided to add a validator and put a default value for clusterId. We cannot use auto-generated Cluster ID because it is saved on a disk by Kafka. If you change the cluster ID variable, it will not be changed on the disk.
This option also allows to avoid cases when somebody deploys 1 replica, then tries to scale the cluster and gets issues because of it.